doc/reference/bluetooth/hci.txt

```diff
@@ -800,6 +800,221 @@ When the Set_Scan_Request_Reports command has completed, a Command Complete
+	|                    |     enabled Advertisement set        |
+	|                    |     Handle                           |
+	|                    |                                      |
+	|                    | Handle_Type_Scan:                    |
+	| 0x0000 - 0xFFFF    |     Scanner Handle (Handle value is  |
+	|                    |     ignored)                         |
+	|                    |                                      |
+	|                    | Handle_Type_Conn:                    |
+	| 0x0000 - 0x0EFF    |     Connection Handle                |
+	+--------------------+--------------------------------------+
+
+	Tx_Power_Level:                                 Size: 1 Octet
+	+--------------------+--------------------------------------+
+	| Value              | Parameter Description                |
+	+--------------------+--------------------------------------+
+	| -127 <= N <= 126   | The desired Tx_Power_Level in signed |
+	|                    | 1 octet integer format to be set to  |
+	|                    | the Handle_Type and Handle pair. The |
+	|                    | controller  shall choose a power     |
+	|                    | level lower than or equal to the one |
+	|                    | specified by the host unless the host|
+	|                    | desired power is lower than the      |
+	|                    | minimum supported Tx power of the    |
+	|                    | controller, in which case the        |
+	|                    | controller shall use its minimum Tx  |
+	|                    | power.                               |
+	|                    | Units: dBm                           |
+	|                    |                                      |
+	| 127                | No preference for the Tx_Power_Level,|
+	|                    | the controller shall revert to using |
+	|                    | its default setting for Tx power.    |
+	+--------------------+--------------------------------------+
+
+	Status:                                         Size: 1 Octet
+	+--------------------+--------------------------------------+
+	| Value              | Parameter Description                |
+	+--------------------+--------------------------------------+
+	| 0x00               | Command succeeded                    |
+	| 0x01-0xFF          | Command failed                       |
+	+--------------------+--------------------------------------+
+
+	Selected_Tx_Power:                              Size: 1 Octet
+	+--------------------+--------------------------------------+
+	| Value              | Parameter Description                |
+	+--------------------+--------------------------------------+
+	| -127 <= N <= 126   | The controller selected Tx power to  |
+	|                    | set for the given Handle_Type and    |
+	|                    | and pair formatted as a 1 octet      |
+	|                    | signed integer. The controller shall |
```

Comment:
"The controller shall not modify this transmit power unless it is directed to do so by the host"
Does this include the No preference value?
I'm assuming that if the host set "no preference" then the controller is free to do dynamic TX power adjustments and not have to commit to the value it returned in this case?

Reply:
No, it does not include the "No preference" value.

If host has no preference than the controller will revert to using the default Tx power as [mentioned](https://github.com/zephyrproject-rtos/zephyr/pull/19779/commits/6012d2b2ca1f5515ee0ae6dfc760301bfc4b0faa#diff-9c37bb9bf44e4a2035f47bde14315da6R897).

I used the formatting and wording model from `7.8.53 LE Set Extended Advertising Parameters Command` in the Core Spec.

Reply:
I will mark this as resolved for now as I don't see any issue here as the best next thing the controller can do if host has no preference is to actually use the defaults :)